README.md

```diff
@@ -8,9 +8,54 @@ Linting and formatting configuration shared by Netlify Node.js repositories:
 - [Editorconfig](https://editorconfig.org/)
 - `.gitattributes`
 
-To add to a new Node.js repository, please check an existing Netlify Node.js repository and copy its:
+## How to add to a new Node.js repository
 
-- `.eslintrc.js`
-- `.prettierrc.json`
-- `.editorconfig`
-- `.gitattributes`
+- `npm install -D @netlify/eslint-config-node husky cross-env npm-run-all`
+- Add the following `.eslintrc.js` at the root. Individual `rules` and `overrides` can be tweaked for the specific
+  project.
+
+```js
+const { overrides } = require('@netlify/eslint-config-node')
+
+module.exports = {
+  extends: '@netlify/eslint-config-node',
+  rules: {},
+  overrides: [...overrides],
+}
+```
+
+- Add the following `.prettierrc.json` at the root:
+
+```json
+"@netlify/eslint-config-node/.prettierrc.json"
+```
+
+- Copy the `.editorconfig` and `.gitattributes` files to the root
```

Comment:
```suggestion
- Copy the `.editorconfig`,`.gitattributes` and `./scripts/run-on-error.js` files to the root
```

Reply:
My suggestion can probably be phrased in a better way

Reply:
Oh I forgot about `run_on_error.js`, right!
I think your phrasing looks good :+1: